modular_skyrat/modules/goofsec/code/sol_fed.dm

```diff
@@ -155,52 +406,134 @@ GLOBAL_LIST_INIT(emergency_responders, list())
 	suit = /obj/item/clothing/suit/armor/riot
 	r_pocket = /obj/item/lighter
 	l_pocket = /obj/item/restraints/handcuffs
-	id = /obj/item/card/id/advanced/black
+	id = /obj/item/card/id/advanced/solgov
 	l_hand = /obj/item/gun/ballistic/shotgun/riot
+	backpack_contents = list(/obj/item/storage/box/survival = 1,
+		/obj/item/storage/box/handcuffs = 1,
+		/obj/item/melee/baton/security/loaded = 1,
+		/obj/item/storage/box/lethalshot = 2,
+		/obj/item/solfed_reporter/treason_reporter = 1,
+		/obj/item/beamout_tool = 1)
+
+	id_trim = /datum/id_trim/solgov
+
+/datum/antagonist/ert/request_911/treason_destroyer
+	name = "Sol Federation Military"
+	role = "Private"
+	department = "Military"
+	outfit = /datum/outfit/request_911/treason_destroyer
+
+/datum/antagonist/ert/request_911/treason_destroyer/greet()
+	var/missiondesc =  ""
+	missiondesc += "<B><font size=5 color=red>You are NOT a Nanotrasen Employee. You work for the Sol Federation as a [role].</font></B>"
+	missiondesc += "<BR>You are here to assume control of [station_name()] due to the occupants engaging in Treason as reported by our SWAT team.\n"
+	missiondesc += "<BR><B>Your Mission</B>:"
+	missiondesc += "<BR> <B>1.</B> Contact the SWAT Team and the First Responders via your cell phone to get the situation from them."
+	missiondesc += "<BR> <B>2.</B> Arrest all suspects involved in the treason attempt."
+	missiondesc += "<BR> <B>3.</B> Assume control of the station for the Sol Federation, and initiate evacuation procedures to get non-offending citizens \
+		away from the scene."
+	missiondesc += "<BR> <B>4.</B> If you need to use lethal force, do so, but only if you must."
+	to_chat(owner, missiondesc)
+	var/mob/living/greeted_mob = owner.current
+	greeted_mob.playsound_local(greeted_mob, 'sound/effects/families_police.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)
+
+/datum/outfit/request_911/treason_destroyer
+	name = "911 Response: SolFed Military"
+
+	uniform = /obj/item/clothing/under/rank/security/officer/hecu
+	head = /obj/item/clothing/head/helmet
+	mask = /obj/item/clothing/mask/gas/hecu2
+	gloves = /obj/item/clothing/gloves/combat
+	suit = /obj/item/clothing/suit/armor/vest
+	shoes = /obj/item/clothing/shoes/combat
+
+	back = /obj/item/storage/backpack/duffelbag/cops
+	glasses = /obj/item/clothing/glasses/sunglasses
+	ears = /obj/item/radio/headset/headset_sec/alt
+	l_pocket = /obj/item/restraints/handcuffs
+	id = /obj/item/card/id/advanced/solgov
+	r_hand = /obj/item/gun/ballistic/automatic/assault_rifle/m16
 	backpack_contents = list(/obj/item/storage/box/handcuffs = 1,
-	/obj/item/melee/baton/security/loaded = 1,
-	/obj/item/storage/box/lethalshot = 2)
+		/obj/item/melee/baton/security/loaded = 1,
+		/obj/item/ammo_box/magazine/m16 = 4)
```

Comment:
Usually it's good practice to put the ) on the line after, it's clearer. But then again I'd argue the whole list should probably not have anything after the initial `list(`, but that's just readability opinions.
```suggestion
		/obj/item/ammo_box/magazine/m16 = 4
	)
```